ci/playbooks/bootstrap-networking-mapper.yml

```diff
@@ -33,6 +33,7 @@
           -e @scenarios/centos-9/base.yml
           -e cifmw_networking_mapper_ifaces_info_path=/etc/ci/env/interfaces-info.yml
           -e "@{{ ansible_user_dir }}/ci-framework-data/artifacts/parameters/zuul-params.yml"
+          -e cifmw_networking_mapper_ifaces_info_path=/etc/ci/env/interfaces-info.yml
```

Comment:
(blocking) Unsure if necessary but I see this duplicated.

Reply:
Yes this is a mistake, I removed the duplicate extra-vars line thank you!